.github/workflows/unit_tests_with_minimum_deps.yml

```diff
@@ -32,6 +32,18 @@ jobs:
       - name: Install featuretools with no dependencies
         run: |
           python -m pip install -e . --no-dependencies
+      - if: ${{ matrix.libraries == 'koalas' }}
+        name: Get numpy version for koalas
+        run: |
+          NUMPY_VERSION=$(cat featuretools/tests/requirement_files/minimum_koalas_requirements.txt | grep numpy)
+          python -m pip uninstall numpy -y
+          python -m pip install $NUMPY_VERSION --no-build-isolation
+      - if: ${{ matrix.libraries == 'core' }}
+        name: Get numpy version for core
```

Comment:
For these 2 names, perhaps "Install numpy version . . ." for each name field?

Reply:
fixed